_layouts/base.html

```diff
@@ -87,75 +89,79 @@
-
-    <header class="usa-header usa-header--extended" role="banner">
-      <div class="usa-navbar">
-        <div class="usa-logo" id="basic-logo">
-          <a href="{{ site.baseurl }}/" title="Home" aria-label="Home">
-            <img src="{{ site.baseurl }}/assets/img/login-gov-logo.svg"
-              class="usa-logo__img"
-              alt="Login.gov"
-              role="img">
-            <em class="usa-logo__text">Developers</em>
-          </a>
-        </div>
-        <button class="usa-menu-btn">Menu</button>
-      </div>
-      <nav role="navigation" class="usa-nav border-top-none">
-        <div class="usa-nav__inner">
-          <button class="usa-nav__close">
-            <img src="{{ site.baseurl }}/assets/img/close.svg" alt="Close" role="img">
-          </button>
-          <div class="usa-nav__secondary">
-            <ul class="usa-nav__secondary-links"></ul>
-            <section aria-label="Search component">
-              {% include search.html id="header-nav" %}
-            </section>
+      <header class="usa-header usa-header--extended" role="banner">
+          <div class="usa-navbar">
+            <div class="usa-logo" id="basic-logo">
+              <a href="{{ site.baseurl }}/" title="Home" aria-label="Home">
+                <img src="{{ site.baseurl }}/assets/img/login-gov-logo.svg"
+                  class="usa-logo__img"
+                  alt="Login.gov"
+                  role="img">
+                <em class="usa-logo__text">Developers</em>
+              </a>
+            </div>
+            <button class="usa-menu-btn">Menu</button>
           </div>
-          <ul class="usa-nav__primary usa-accordion">
-            {% include nav/list.html
-              links = site.data.nav.primary
-              li_class = 'usa-nav__primary-item'
-              subnav_ul_class = 'desktop:display-none usa-sidenav__sublist'
-            %}
-          </ul>
-        </div>
-      </nav>
-    </header>
+          <nav role="navigation" class="usa-nav border-top-none">
+            <div class="usa-nav__inner">
+              <button class="usa-nav__close">
+                <img src="{{ site.baseurl }}/assets/img/close.svg" alt="Close" role="img">
+              </button>
+              <div class="usa-nav__secondary">
+                <ul class="usa-nav__secondary-links"></ul>
+                <section aria-label="Search component">
+                  {% include search.html id="header-nav" %}
+                </section>
+              </div>
+              <ul class="usa-nav__primary usa-accordion">
+                {% include nav/list.html
+                  links = site.data.nav.primary
+                  li_class = 'usa-nav__primary-item'
+                  subnav_ul_class = 'desktop:display-none usa-sidenav__sublist'
+                %}
+              </ul>
+            </div>
+          </nav>
+      </header>
+      {% if page.sidenav %}
+      <header class="usa-dark-background usa-section--dark">
```

Comment:
The addition of a separate `<header>` here will likely trip some accessibility tools, since a screen reader user who navigates using landmark regions will have trouble understanding the difference between the two header sections. Could we assign an `aria-label` to one or both of these?

Example in Axe browser extension:

![image](https://github.com/18F/identity-dev-docs/assets/1779930/3dffc5db-e928-4b04-b538-0bec2ca2cd7a)

Reference: https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=AxeChrome